airbyte-cdk/bulk/toolkits/load-iceberg-parquet/src/main/kotlin/io/airbyte/cdk/load/command/iceberg/parquet/IcebergCatalogSpecifications.kt

```diff
@@ -257,6 +289,19 @@ data class NessieCatalogConfiguration(
     val accessToken: String?
 ) : CatalogConfiguration
 
+/**
+ * Rest catalog configuration details.
+ *
+ * Stores information required to connect to a Rest server.
+ */
+@JsonSchemaTitle("Nessie Catalog Configuration")
```

Comment:
```suggestion
@JsonSchemaTitle("REST Catalog Configuration")
```

(plus the next few lines)